.github/workflows/build.yml

```diff
@@ -2,14 +2,61 @@ name: Pipeline
 
 on: [push, pull_request]
 
+env:
+  GRADLE_OPTS: "-Dorg.gradle.internal.launcher.welcomeMessageEnabled=false"
+
 jobs:
-  build:
-    name: "Build, Test and Publish"
-    runs-on: ubuntu-latest
+  test-matrix:
```

Comment:
We should add a name for this job - something like `Test (${{matrix.ox}})` perhaps?

Reply:
Sound good, will do